src/components/dock/dock.types.ts

```diff
@@ -40,6 +40,11 @@ export interface DockItem {
      * Used to specify a custom component to render as a menu for the dock item.
      */
     dockMenu?: DockMenu;
+
+    /**
+     * If specified, will display a notification badge on the buttons in the dock.
```

Comment:
how do you mean "other than 0"?

Reply:
Sorry, I missed this. Currently, the badge will not be shown when `badge === 0`. Because 0 is a falsy value when you check

```
if (this.item.badge) {}